hackathon_site/hardware/test_api.py

```diff
@@ -194,3 +194,152 @@ def test_hardware_get_success(self):
         data = response.json()
 
         self.assertEqual(expected_response, data["results"])
+
+
+class HardwareListTestCase(SetupUserMixin, APITestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.hardware1 = Hardware.objects.create(
+            name="hardware1",
+            model_number="model",
+            manufacturer="manufacturer",
+            datasheet="/datasheet/location/",
+            notes="notes",
+            quantity_available=0,
```

Comment:
Since you're creating `OrderItems` for this below, you should set `quantity_available=1`. 

Otherwise it's not legal to create an order item for this hardware, so there could be unintended side effects in the future.